Sekoia.io/sekoiaio/triggers/intelligence.py

```diff
@@ -114,10 +114,10 @@ def next_batch(self):
 
             event_name = f"Sekoia.io feed - batch of {len(objects)} objects"
             if self.to_file:
-                filepath = write("stix_objects.json", {"stix_objects": objects})
+                filepath = write(self.__class__.FILE_NAME, {"stix_objects": objects})
```

Comment:
I think you should directly save the list of objects or the next actions will have a dict instead of an expected list.
Also `__class__` is not needed

```suggestion
                filepath = write(self.FILE_NAME, objects)
```